WordPressAuthenticator.xcodeproj/project.pbxproj

```diff
@@ -158,6 +158,7 @@
 		F12F9FB424D8A68E00771BCE /* AuthenticatorAnalyticsTracker.swift in Sources */ = {isa = PBXBuildFile; fileRef = F12F9FB324D8A68E00771BCE /* AuthenticatorAnalyticsTracker.swift */; };
 		F12F9FB824D8A7FC00771BCE /* AnalyticsTrackerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = F12F9FB724D8A7FC00771BCE /* AnalyticsTrackerTests.swift */; };
 		F180B82424F59263000A01F5 /* StoredCredentialsPicker.swift in Sources */ = {isa = PBXBuildFile; fileRef = F180B82324F59263000A01F5 /* StoredCredentialsPicker.swift */; };
+		F18DF0E5252500A600D83AFE /* WordPressAuthenticatorTests-Bridging-Header.h in Sources */ = {isa = PBXBuildFile; fileRef = F18DF0E4252500A600D83AFE /* WordPressAuthenticatorTests-Bridging-Header.h */; };
```

Comment:
The reason why there are several project changes is that I needed to call some Swift code from ObjC, and our tests target was failing to recognize the Swift symbols, until I added a bridging header AND and interface header to the testing target.  Please let me know if you want more details.